test/approvals/_dcuc_logo_data_work_protectli_vault_cml_v1_2_0_rc1_vp46xx_rom_l_data_work_bootsplash_bmp

```diff
@@ -0,0 +1,2 @@
+Setting /home/macpijan/projects/github/dasharo/dcu/test/data/work/bootsplash.bmp as custom logo
```

Comment:
Is it fine to test against the full path on your device in here? The result will be different depending on who and on what machine performs it. Maybe the approval file should look something like `Setting ${PWD}/test.data/(...)`? In the end every time anyone with different path to the dcu repo will test it, the approval would need to be done manually in both cases. This refers to all the other approval files with an absolute path.

Reply:
Oh, the approval files are not compared against. The are generates as a proof of the approval. Then it's fine.

Reply:
> Is it fine to test against the full path on your device in here?

I guess not

> Oh, the approval files are not compared against. The are generates as a proof of the approval. 

I think they should be compared. That is the point of this tool. Can you share the full log of test run on your side?

There is an option to allow for some diff (such as local paths): https://github.com/dannyben/approvals.bash?tab=readme-ov-file#allowing-some-difference

Reply:
Sorry, I still had to figure out how those tests actually work. I had wrong assumptions about that.